Lib/urllib/parse.py

```diff
@@ -469,6 +469,9 @@ def urlsplit(url, scheme='', allow_fragments=True):
         else:
             scheme, url = url[:i].lower(), url[i+1:]
 
+    _unsafe_chars_to_remove = ['\t', '\r', '\n']
```

Comment:
Setting this as a module level attribute helps some users override the variable as needed. Last time a similar security fix was done cherrypy wanted to override the check : https://bugs.python.org/issue36274

Reply:
Done.  It's a process wide global so I hope no library or frameworks do that lightly... lest they introduce bugs in other places.  But we're keeping it _ prefixed for a reason, they know there be dragons. :)